README.md

```diff
@@ -251,6 +251,10 @@ To request automatic tracing support for a module not on this list, please [file
 
 - All plugins have been removed in favor of instrumentations.
 
+- `HttpBaggage` renamed to `HttpBaggagePropagator`
```

Comment:
I think 0.19.0 has been released already (at least to NPM, tag + github are missing) so a new section should be created here.